src/explorer/Metrics/CategoricalData.cs

```diff
@@ -0,0 +1,26 @@
+namespace Explorer.Metrics
+{
+    using System.Collections.Generic;
+    using System.Text.Json;
+
+    public class CategoricalData
+    {
+        public CategoricalData(ValuesListType values, ValueCounts valueCounts)
+        {
+            Values = values;
+            ValueCounts = valueCounts;
+        }
+
+        public ValuesListType Values { get; }
+
+        public ValueCounts ValueCounts { get; }
+
+        public class ValuesListType : List<ValueWithCount<JsonElement>>
+        {
+            public ValuesListType(IEnumerable<ValueWithCount<JsonElement>> items)
+                : base(items)
+            {
+            }
+        }
+    }
```

Comment:
I'm not sure this is a good pattern to use - ie. having an inner class that needs to be instantiated and passed from the outside. 

IMO with simple classes like this that just wrap some data, it's better to pass the data and encapsulate the 'wrapping' part within the wrapper class. 

So instead of using it like this:
```csharp
var data = new CategoricalData(new CategoricalData.ValuesListType(values)), valueCounts);
```
It would be used like this:
```csharp
var data = new CategoricalData(values, valueCounts);
```

And then the `ValuesListType` creation happens in the constructor:
```csharp
        public CategoricalData(IEnumerable<ValueWithCount<JsonElement>> items values, ValueCounts valueCounts)
        {
            Values = new ValuesListType(values);
            ValueCounts = valueCounts;
        }
```

In fact, in this particular case, I'm not sure there's any need for an inner class since it doesn't add any new functionality - it's basically an alias for `List<ValueWithCount<JsonElement>>` ? Why not just use the `List<...>` type?

Reply:
fixed with cc77143